packages/widgets/src/lib/components/InferenceWidget/InferenceWidget.svelte

```diff
@@ -85,6 +86,8 @@
 			  ? WIDGET_COMPONENTS[model.pipeline_tag as keyof typeof WIDGET_COMPONENTS]
 			  : undefined;
 
+	$isLoggedInStore = isLoggedIn;
```

Comment:
The widget is not picking this up in the tasks page (I'm logged in but the popover still shows)
(needs fixing in moon-landing probably)

![image](https://github.com/huggingface/huggingface.js/assets/33657802/79327ab4-bfa5-4c8d-b92b-e40d8c17e685)